### DISCUSSION TITLE
No "Approve and run" button showing up on a PR from a public fork

### DISCUSSION THREAD

**Discussion by elias-pap** [community/community#119459] [Pull Requests] [answered]
### Select Topic Area

Bug

### Body

No "Approve and run" button on this PR: https://github.com/elias-pap/youtube-share-clip/pull/60

**Comment by elias-pap** [comment]
According to step 5 from this section of the docs: https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks#approving-workflow-runs-on-a-pull-request-from-a-public-fork

"If you are comfortable with running workflows on the pull request branch, return to the Conversation tab, and under "Workflow(s) awaiting approval", click Approve and run."

There should be an "Approve and run" button on Conversation tab, but there isn't one.

**Comment by elias-pap** [comment, marked as answer]
Usage error. The workflow was set to be triggered on `push` event but this does not work for PRs from forks. Changed it to be triggered on `pull_request` event.

**Comment by dlyssenko** [comment]
I have this issue too. 
Bizarre enough when I submit a PR from the fork that I pulled (as a maintainer) - the CI runs automatically and I can trigger it manually too. But when the PR comes from the external fork - there's no way to run the CI workflow (and there's no button like "Approve and run" - I can only approve the PR from the "File" tab.

Here's my trigger:
`on: [ pull_request, workflow_dispatch, pull_request_target ]`
